### DISCUSSION TITLE
SSL certificate for suricata integration on port 5055

### DISCUSSION THREAD

**Discussion by k-alki** [Security-Onion-Solutions/securityonion#11385] [2.4]
### Version

2.4.10

### Installation Method

Network installation on Ubuntu

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

250Go

### Storage for /nsm

50

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi !

Which certificate should i use for using the suricata integration ? More generally, which certificate authority can i fingerprint for the port 5055 ?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
What exactly are you trying to do?

Is this related to your other discussions at https://github.com/Security-Onion-Solutions/securityonion/discussions/11378 and https://github.com/Security-Onion-Solutions/securityonion/discussions/11383?
